ocrd_monitor/docker-compose.yml

```diff
@@ -12,9 +12,28 @@ services:
 
     hostname: ${MONITOR_HOST}
 
+    environment:
+      - "CONTROLLER=${CONTROLLER_HOST}:${CONTROLLER_SSH_PORT}"
+
     ports:
       - ${MONITOR_PORT_WEB}:8080
       - ${MONITOR_PORT_GTK}:8085
 
     volumes:
       - ${MONITOR_DATA}:/data
+      - ${MANAGER_KEY}:/id_rsa
+      - shared:/run/lock/ocrd.jobs
+
+  dozzle:
+    image: amir20/dozzle:latest
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Is this necessary? If yes we should add a env variable for other environments of host system.

Reply:
Yes, this is essential. It gives the container access to the Docker daemon.

If non-Unix hosts use a different location, then we must try to find a more general formulation.

Reply:
Yes in my case it is not possible.

```
Error response from daemon: _modules\ocrd_manager\ocrd_monitor\var\run\docker.sock%!(EXTRA string=is not a valid Windows path)
make: *** [Makefile:62: start] Error 1

```

Reply:
Adding another slash as first character of host path seems to work on windows environments.

https://stackoverflow.com/questions/36765138/bind-to-docker-socket-on-windows/58204900#58204900

Reply:
Feel free to modify the branch!